lib/helper/Puppeteer.js

```diff
@@ -749,7 +749,8 @@ class Puppeteer extends Helper {
     if (locator) {
       const els = await this._locate(locator);
       assertElementExists(els, locator, 'Element');
-      await els[0]._scrollIntoViewIfNeeded();
```

Comment:
how about doing try-catch or checking for this method to support Puppeteer <= 14 ?

Reply:
I guess with new change `await el.evaluate((el) => el.scrollIntoView());` that is backward compatibility.